### PR TITLE
chore(a11y): adds some missing labels around inputs

### LINKED ISSUE #5712: Make sure all controls are associated with labels
The following are currently missing labels:
 * [x] Username and password fields on login form
 * [x] River selector
 * [x] Category filter on admin/plugins page
 * [x] Sort dropdown on admin/plugins page

This is not an exhaustive list.

### PR DESCRIPTION
**Pull request by jeabakker**
fixes #5712

TODOs:
- [x] Add docs to "upgrading" guide that mention this change.

### CODE REVIEW

**Comment by mrclay** [comment]
Will this affect any plugins (does it need a mention in docs/guides/upgrading)? /cc @ewinslow

**Comment by jeabakker** [comment]
It'll affect theme's as they are most likely to take over the login form

File: views/default/forms/admin/plugins/sort.php [hunk 1]
```diff
@@ -8,7 +8,9 @@
  */
 
 echo '<div class="mtm">';
+echo '<label for="admin-plugins-sort" class="hidden">' . elgg_echo('sort') . '</label>';
```
Comment (ewinslow) [inline views/default/forms/admin/plugins/sort.php]: Use the "hidden" attribute instead of the class

**Comment by ewinslow** [comment]
This is great but feels like a spot-fix. There are tons of forms throughout Elgg. Are we doomed to fix them one by one?

File: views/default/forms/admin/plugins/sort.php (hunk 1 above)
Comment (jeabakker) [inline views/default/forms/admin/plugins/sort.php]: like with many things IE is lagging behind on support of the hidden attribute (source http://www.w3schools.com/tags/att_global_hidden.asp)
what is the support policy for browsers?

**Comment by jeabakker** [comment]
> This is great but feels like a spot-fix. There are tons of forms throughout Elgg. Are we doomed to fix them one by one?

yes :(

File: views/default/forms/admin/plugins/sort.php (hunk 1 above)
Comment (ewinslow) [inline views/default/forms/admin/plugins/sort.php]: http://learn.elgg.org/en/latest/intro/install.html#browser-support-policy

Comment (ewinslow) [inline views/default/forms/admin/plugins/sort.php]: It is pretty easily shimmed with "[hidden] { display: none !important }" somewhere in the code, no?

Comment (jeabakker) [inline views/default/forms/admin/plugins/sort.php]: what would be the best css file to add the shim to? css/elements/reset?

Comment (ewinslow) [inline views/default/forms/admin/plugins/sort.php]: SGTM

**Comment by jeabakker** [comment]
updated pr with supplied comments

**Pull request merged**